bots/task/github.py

```diff
@@ -183,6 +183,11 @@ def request(self, method, resource, data="", headers=None):
                 self.conn.request(method, self.qualify(resource), data, headers)
                 response = self.conn.getresponse()
                 break
+            # This happens when TLS is the source of a disconnection
+            except socket.error as ex:
+                if connected or ex[0] != errno.EPIPE:
```

Comment:
`ex[0]` is a bit magic, how about `ex.errno`? (Just verified that this works as well)

Reply:
Updated to use errno.